### ISSUE TITLE
stop skidding mf

### ISSUE THREAD

**Issue by SarventeFnf** [LeakingAnomalyAppsSources/LiteNiteV2Backend#3]
your a little skid, Your the reason why Two wrongs doesnt make a right, your the reason on why slavery created, you little fucking skid, go get burned by satan you failed child, mother was on her period while she having you, she probs looked at you and syytaid, "this google light mode child is horror looking" fucking skid

**Comment by LeakingAnomalyAppsSources** [comment]
BHAHAH. Firstly your a skid here for working for Stormzy. Secondly you don't like Fexor really you're just pitying him. He don't love you. You have no life and should be satan himself. Bye kiddo

**Comment by LeakingAnomalyAppsSources** [comment]
I wanna have some questions answered.
Do you really Love Fexor? 
Does he really Love you?

**Comment by SarventeFnf** [comment]
you def slax, or Vinz, one of the two. also i love fexor, you just mad that you leak sources so other people can skid, your the skidder here fat fuck learn how to go sit yo ass down

**Comment by LeakingAnomalyAppsSources** [comment]
LOL I bet you don't love him :)

**Comment by LeakingAnomalyAppsSources** [comment]
Fine prove it

**Comment by SarventeFnf** [comment]
already proved it, 34 mins and it be our time you failed abortion

**Comment by LeakingAnomalyAppsSources** [comment]
your time to what?

**Comment by SarventeFnf** [comment]
You'll see, maybe if you was here you would see what would happen, Oh wait your banned

**Comment by LeakingAnomalyAppsSources** [comment]
tell me now. or when the time happens

**Comment by SarventeFnf** [comment]
Nope, u talked all that shit, so i aint tellin, maybe you'll find out soon

**Comment by LeakingAnomalyAppsSources** [comment]
I have alts in the server you know. Are you like going to put it in news?

**Comment by SarventeFnf** [comment]
Idk maybe not, maybe will u tell me :I

**Comment by LeakingAnomalyAppsSources** [comment]
what do you mean?

**Comment by SarventeFnf** [comment]
Der Tod wartet hm..

**Comment by LeakingAnomalyAppsSources** [comment]
tf?

**Comment by LeakingAnomalyAppsSources** [comment]
we'll find out in 10 minutes

**Comment by LeakingAnomalyAppsSources** [comment]
Congrats your dating Fexor and V3 that I am going to leak

**Comment by SarventeFnf** [comment]
Pls its not even out, you aint gonna do shit. do it, and im reporting you to github for skidding, and being a threat to discord

**Comment by LeakingAnomalyAppsSources** [comment]
oh honey please

**Comment by SarventeFnf** [comment]
''honey''
